worker/worker/request_handlers.go

```diff
@@ -241,6 +241,11 @@ func HandleStreamRequest(request *pb.StreamRequest) {
 	S.endTranscoding(streamCtx.getStreamName())
 	notifyTranscodingDone(streamCtx)
 
+	err = transcodeAudio(streamCtx)
```

Comment:
I think we should only extract the audio of the "COMB" stream, otherwise we'll end up with 3 audio tracks of the same lecture.

Reply:
Agree.

Reply:
For now, I think we can go with this. When we refactor the worker, we can implement a way to figure out which version of the stream is the main one.

Reply:
I think it would be quite cool to specify a custom "pipeline"  (a custom function) for each version of a stream. That way we could potentially add new versions quite easily.